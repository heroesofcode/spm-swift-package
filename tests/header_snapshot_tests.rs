use insta::assert_snapshot;
use colored;
use spm_swift_package::Header;

#[test]
fn test_show_header_snapshot() {
    colored::control::set_override(false);
    let header = Header::show_header();
    assert_snapshot!(header);
}