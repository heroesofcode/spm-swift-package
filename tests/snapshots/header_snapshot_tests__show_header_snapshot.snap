---
source: tests/header_snapshot_tests.rs
expression: header
snapshot_kind: text
---
SPM Swift Package
ğŸš€ You can create your Swift Package via the command line ğŸ”¨
v0.4.1
